x-pack/plugins/spaces/public/management/view_space/view_space.tsx

```diff
@@ -139,27 +146,9 @@ export const ViewSpacePage: FC<PageProps> = (props) => {
   }
 
   const HeaderAvatar = () => {
-    return space.imageUrl != null ? (
-      <Suspense fallback={<EuiLoadingSpinner />}>
-        <LazySpaceAvatar
-          space={{
-            ...space,
-            initials: space.initials ?? 'X',
-            name: undefined,
-          }}
-          size="xl"
-        />
-      </Suspense>
-    ) : (
+    return (
       <Suspense fallback={<EuiLoadingSpinner />}>
-        <LazySpaceAvatar
-          space={{
-            ...space,
-            name: space.name ?? 'Y',
-            imageUrl: undefined,
-          }}
-          size="xl"
-        />
```

Comment:
We don't need this conditional, the component itself already contains logic to handle whatever is missing